x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_badges.tsx

```diff
@@ -18,17 +18,17 @@ export const DataStreamsBadges: React.FunctionComponent<Props> = ({ dataStream }
   const badges = [];
   if (isFleetManaged(dataStream)) {
     badges.push(
-      <EuiBadge color="hollow">
+      <EuiBadge color="hollow" key={'managed'}>
```

Comment:
The `key` prop is for React warning about unique prop for lists

Reply:
👍 thanks for fixing this!